doc/source/whatsnew/v1.1.0.rst

```diff
@@ -117,10 +117,10 @@ compatibility (:issue:`3729`)
 .. ipython:: python
 
     # Default ``dropna`` is set to True, which will exclude NaNs in keys
-    df_dropna.groupby(by=["b"], dropna=True).sum()
+    df_dropna.groupby(by="b", dropna=True).sum()
```

Comment:
This deprecation should have no impact on this result, and so you should not need to make modifications such as these. Can you revert these (and other similar) changes.

Reply:
@rhshadrach  Done